### PR TITLE
Add validation rules to Generator

### PR DESCRIPTION
**Pull request by WinterSilence**
| Q             | A
| ------------- | ---
| Is bugfix?    | ✔️
| New feature?  | ❌
| Breaks BC?    | ❌
| Fixed issues  |

### CODE REVIEW

File: src/Generator.php [hunk 1]
```diff
@@ -212,6 +212,16 @@ public function rules()
         return [
             [['template'], 'required', 'message' => 'A code template must be selected.'],
             [['template'], 'validateTemplate'],
+            [['enableI18N'], 'boolean'],
+            [
+                ['messageCategory'],
+                'match',
+                'pattern' => '/^[-\w\/]+$/',
```
Comment (samdark) [inline src/Generator.php:219]: There are no such limitations on message category.

**Review by samdark — CHANGES_REQUESTED** [review]

File: src/Generator.php (hunk 1 above)
Comment (WinterSilence) [reply src/Generator.php]: @samdark replaced to `string` validation

**Comment by WinterSilence** [comment]
~~~php
$var = 1;
~~~

**Review by samdark — APPROVED** [review]

**Comment by samdark** [comment]
Please add a CHANGELOG line.

**Comment by samdark** [comment]
Thanks.

**Pull request merged**